src/logic/filter-params.ts

```diff
@@ -116,6 +107,7 @@ export function extractParams(url: URL) {
     : [0, 0]
   const center = { x, y }
   const showOnSale = url.searchParams.get('on-sale') === 'true'
+  const showListedForRent = url.searchParams.get('listed-for-rent') === 'true'
```

Comment:
this is nip ticking.. but to me it would sound more consistent to call these either `on-sale` & `on-rent` or `listed-for-sale` & `listed-for-rent` than having  them with different wording.. same for the variables (showOnSale, showListedForRent, etc). Wdyt?

Reply:
I completely agree, but the `on-rent` phrasing is confusing as the tile is not on rent but is `for rent` or is `listed for rent`. I didn't change the `on-sale` one to not add a breaking change. How do you think we should proceed with the name?

Reply:
okay, yea i would not break the api now, this was a detail